src/components/Components/ComponentDetails/tabs/ComponentLatestBuild.tsx

```diff
@@ -45,7 +44,9 @@ const ComponentLatestBuild: React.FC<React.PropsWithChildren<ComponentLatestBuil
   const [taskRuns, taskRunsLoaded] = useTaskRuns(namespace, pipelineRun?.metadata?.name);
   const buildLogsModal = useBuildLogViewerModal(component);
 
-  const containerImage = getLastestImage(component);
+  // Avoid getLastestImage fallback to spec.containerImage, which lacks image tag
+  // and causes 'cosign download sbom' to fail. Use lastPromotedImage explicitly.
+  const containerImage = component?.status?.lastPromotedImage;
```

Comment:
maybe,  let us fix the problem of getLatestImage directly? just removing the fallback from that function?

Reply:
I thought about it, but the thing is that in certain scenarios having the fallback works just fine. For example:

https://github.com/user-attachments/assets/520ad10c-bf13-4de1-8f3b-cc2c9778ed42

in those examples it's using the `spec.containerImage` (fallback) and it works fine, because it simply open the quay.io repo 🙂

the issue with the SBOM is that the command fails without the image tag, so it would fail every time it uses the `spec.containerImage` fallback